lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -111,6 +113,27 @@ class PolicyElementAssociation < ::ActiveRecord::Base
       has_and_belongs_to_many :operations, class_name: :"PolicyMachineStorageAdapter::ActiveRecord::Operation"
       belongs_to :user_attribute
       belongs_to :object_attribute
+
+      #TODO: ActiveRecord's generated operations= method is inefficient, makes 1 query for each op added or removed even though there's no hooks
+      # Awkward manual implementation for now, but in the future change this to an hstore or something in the postgres adapter,
+      # and/or fix Rails.
+      def operations=(updated_operations)
```

Comment:
Should there actually be additional specs to make sure this method behaves correctly